package.json

```diff
@@ -53,13 +53,13 @@
     "@types/jest": "24.0.12",
     "@types/lodash": "4.14.123",
     "@types/node": "12.6.2",
-    "jest": "24.5.0",
```

Comment:
Is it required to upgrade `jest`?

I feel it's a breaking change for the suite, as in current  major support for Node.js v10 must be maintained and we should be able to run tests with that version

Reply:
Good catch, seems like the latest version of `jest` doesn't support Node.js v10. There is just a warning that typescript version is not officially supported:
```
ts-jest[versions] (WARN) Version 4.7.4 of typescript installed has not been tested with ts-jest. If you're experiencing issues, consider using a supported version (>=2.7.0 <4.0.0). Please do not report issues in ts-jest if you are using unsupported versions.
```

Reply:
Also, in case you were wondering why `typescript` was updated. It does not compile with `3.9.*`:
```
node_modules/@sinclair/typebox/typebox.d.ts:66:102 - error TS2574: A rest element type must be an array type.

66 export declare type StaticContructorParameters<T extends readonly TSchema[], P extends unknown[]> = [...{
                                                                                                        ~~~~
67     [K in keyof T]: T[K] extends TSchema ? Static<T[K], P> : never;
   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
68 }];
   ~

node_modules/@sinclair/typebox/typebox.d.ts:87:100 - error TS2574: A rest element type must be an array type.

87 export declare type StaticFunctionParameters<T extends readonly TSchema[], P extends unknown[]> = [...{
                                                                                                      ~~~~
88     [K in keyof T]: T[K] extends TSchema ? Static<T[K], P> : never;
   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
89 }];
   ~

node_modules/@sinclair/typebox/typebox.d.ts:105:21 - error TS2456: Type alias 'IntersectReduce' circularly references itself.

105 export declare type IntersectReduce<I extends unknown, T extends readonly any[]> = T extends [infer A, ...infer B] ? IntersectReduce<I & A, B> : I extends object ? I : {};
                        ~~~~~~~~~~~~~~~

node_modules/@sinclair/typebox/typebox.d.ts:105:104 - error TS2574: A rest element type must be an array type.

105 export declare type IntersectReduce<I extends unknown, T extends readonly any[]> = T extends [infer A, ...infer B] ? IntersectReduce<I & A, B> : I extends object ? I : {};
                                                                                                           ~~~~~~~~~~

node_modules/@sinclair/typebox/typebox.d.ts:105:118 - error TS2315: Type 'IntersectReduce' is not generic.

105 export declare type IntersectReduce<I extends unknown, T extends readonly any[]> = T extends [infer A, ...infer B] ? IntersectReduce<I & A, B> : I extends object ? I : {};
...
```